Lib/booleanOperations/booleanGlyph.py

```diff
@@ -78,7 +82,7 @@ def _get_clockwise(self):
 
     def _get_bounds(self):
         if self._bounds is None:
-            from robofab.pens.boundsPen import BoundsPen
+            from fontTools.pens.boundsPen import BoundsPen
```

Comment:
Sounds good, though I don't know if upstream wants to depend on a bleeding-edge version of fontTools. Maybe we can dispatch that import as well, prioritizing robofab's pen.

By the way, why does fontTools take anchors into account? Is that useful in a certain context?